ztp/tools/policy-object-template-diff/main.go

```diff
@@ -0,0 +1,447 @@
+package main
+
+import (
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	sriov "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	performanceprofile "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v1"
+	tuned "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
+	ptp "github.com/openshift/ptp-operator/api/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/conversion"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
+	"log"
+	configurationPolicyv1 "open-cluster-management.io/config-policy-controller/api/v1"
+	policyv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	"os"
+	"path"
+	"runtime"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+var (
+	sch    = k8sruntime.NewScheme()
+	_      = configurationPolicyv1.AddToScheme(sch)
+	_      = policyv1.AddToScheme(sch)
+	decode = serializer.NewCodecFactory(sch).UniversalDeserializer().Decode
+)
+
+type CR struct {
+	u        unstructured.Unstructured
+	filePath string
+	hasMatch bool
+}
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	setA := make(map[string][]CR)
+
+	setB := make(map[string][]CR)
+
+	/*
+		process set A
+	*/
+	populateMap(setA, getenv("A_PATH", "./example/acmpolicy"))
```

Comment:
Why `getenv` and not command line args?

Reply:
No reason other than it was just faster to churn it out and lack of attention to UX :/ . If it's ok can we pin this (use CLI as input) as a feature of when this becomes something legit? Also would give me some time to playaround with Cobra and Viper